drivers/Makefile.dep

```diff
@@ -36,6 +36,23 @@ ifneq (,$(filter at,$(USEMODULE)))
   USEMODULE += isrpipe_read_timeout
 endif
 
+ifneq (,$(filter at24c128,$(USEMODULE)))
+  USEMODULE += at24cxxx
+endif
```

Comment:
You could match against `at24c%` and handle all the dependency tracking in one `ifneq` ... `endif`

Reply:
Ok

Reply:
You have previously asked me to add the PSEUDOMODULES "at24c256%" and "at24c128".
But they are not really doing anything so far and if someone writes `USEMODULE += at24c256` in his Makefile, it clarifies that he uses at least one at24c256 EEPROM. Or should those PSEUDOMODULES define the constant AT24CXXX_PARAM_EEPROM size?

Reply:
OK, the idea is that you provide the modules `at24cxxx`, `at24c128`, `at24c256`, and `at24c256c`. Only `at24cxxx` is a "real" module, the rest are pseudo modules. By using `at24c256%` in `makefiles/pesudomodules.inc.mk` both `at24c256` and `at24c256c` are matched (as well as everything else with the prefix `at24c256`, but that is an implementation detail the use must not rely on).

Here you are managing the dependencies of all those modules. Ther use is expected to only add one of `at24c128`, `at24c256`, or `at24c256c`. All these would match with the following rule:

```Makefile
ifneq (,$(filter at24c%, $(USEMODULE)))
  FEATURES_REQUIRED += periph_i2c
  USEMODULE += xtimer
  USEMODULE += at24cxxx
endif
```

(You do not need to worry about possible adding `at24cxxx` more than once to `USEMODULE`. The array is sorted and duplicates are filtered out by the build system. In fact RIOT build system runs the dependency checking code recursively until `USEMODULES` (after sorting and filtering for duplicates) no longer changes to catch transient dependencies.)

So the point of this remark is to simply the dependency tracking and not to change the provided (pseudo-)modules or their names.